README.md

```diff
@@ -1,72 +1,192 @@
-http://api.jsdelivr.com/v1/bootstrap/libraries
-http://api.jsdelivr.com/v1/jquery/libraries
+{
+  "name": <libraryName>,
+  "mainfile": <libraryMainFile>,
+  "lastversion": <currentLibraryVersion>,
+  "description": <libraryDescription | "">
+  "homepage": <libraryHomepageURL | "">,
+  "github": <libraryGihubURL | "">,
+  "author": <libraryAuthor | "">,
+  "versions": <arrayOfLibraryVersions>,
+  "meta": {
+    "created": <apiLibraryCreationTime>,
+    "updated": <apiLibraryUpdatedTime | undefined>
+  }
+}
 ```
 
+e.g.
+```
+api.jsdelivr.com/v2/jsdelivr/library/jquery
 
-Get full information for a single library based on `name` parameter.
+// =>
+{
+  "name": "jquery",
+  "mainfile": "jquery.min.js",
+  "lastversion": "2.1.4",
+  "description": "jQuery is a fast and concise JavaScript Library that simplifies HTML document traversing, event handling, animating, and Ajax interactions for rapid web development. jQuery is designed to change the way that you write JavaScript.",
+  "homepage": "http://jquery.com/",
+  "github": "https://github.com/jquery/jquery",
+  "author": "jQuery Foundation",
+  "versions": [
+    "2.1.4",
+    "2.1.3",
+    ...
+  ],
+  "meta": {
+    "created": 1436126637972,
+  }
+}
+```
+
+#### `error`
+
+Should you ever be so unlucky, an error response is as follows
 
 ```
-http://api.jsdelivr.com/v1/jsdelivr/libraries?name=jquery
-http://api.jsdelivr.com/v1/jsdelivr/libraries/jquery - alias
+{
+  "status": <httpStatusCode>,
+  "message": <errorMessage>
+}
 ```
 
-Get full information for any library starting with `jq` that has lastversion ending with `0.1`. [minimatch](https://www.npmjs.org/package/minimatch) syntax is supported.
+e.g.
 
 ```
-http://api.jsdelivr.com/v1/jsdelivr/libraries?name=jq*&lastversion=*.0.1
+api.jsdelivr.com/v2/foo
+
+// =>
+{
+  "status": 404,
+  "message": "Requested url /v2/foo not found."
+}
 ```
 
-You can use any of the following parameters to search for libraries. A search will be performed for projects matching your input. You can use multiple parameters at the same time. If multiple projects match they all will be outputed.
+## Endpoints
 
-* `name` - name of library. Example: jquery
-* `mainfile` - mainfile parameter in info.ini. Example: jquery.min.js
-* `lastversion`- lastversion of the project. Example: 2.0.3 (will match multiple projects)
-* `versions` -  all hosted versions for selected project. (read only)
-* `description` - description of the project
-* `homepage`- webpage of project. Example: http://jquery.com/
-* `github`- github page of project. Example: https://github.com/jquery/jquery
-* `author` - the author of project. Example: jQuery Foundation
-* `assets` - files hosted per versions. (read only)
+Where `<cdn> == jsdelivr | cdnjs | google | bootstrap`,
+`<library>` is a library provided by `<cdn>`
+and `<version>` is some version of `<library>`.
 
+- `/<cdn>/libraries`
+  - Returns an array of library objects for all libraries hosted by <cdn>;
```

Comment:
Its not, its empty https://dl.dropboxusercontent.com/s/zqh9wziki7exhpz/firefox_2016-01-21_23-25-25.png?dl=0